### PR TITLE
Release issues

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Comment by sbrunner** [comment]
Ready to be reviewed

File: c2cgeoportal/lib/lingua_extractor.py [hunk 1]
```diff
@@ -92,6 +96,8 @@ def __call__(self, filename, options):
         self.capabilities_cache = {}
 
         self.env = bootstrap(filename)
+        with open("project.yaml") as f:
+            self.package = yaml.load(f.read())
```
Comment (pvalsecc) [inline c2cgeoportal/lib/lingua_extractor.py]: `yaml.load(f)` is enough.

File: c2cgeoportal/lib/lingua_extractor.py [hunk 2]
```diff
@@ -54,9 +54,13 @@ def __call__(self, filename, options):
         message_str = subprocess.check_output(["node", "tools/extract-messages.js", filename])
         try:
             messages = loads(message_str)
-            return [Message(
-                None, message, None, [], u"", u"", context.split(":")
-            ) for context, message in messages]
+            messages = []
```
Comment (pvalsecc) [inline c2cgeoportal/lib/lingua_extractor.py]: Why do you leave line 56?

**Comment by pvalsecc** [comment]
I'll need you to sit with me for this review...

File: c2cgeoportal/lib/lingua_extractor.py (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/lib/lingua_extractor.py]: Effectively ..

File: c2cgeoportal/lib/lingua_extractor.py (hunk 1 above)
Comment (sbrunner) [inline c2cgeoportal/lib/lingua_extractor.py]: Effectively :-)

**Comment by pvalsecc** [comment]
Ship it

**Pull request merged**